uwsgi/run.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/bash
 
+build_front_end() {
+   cd /app/ui
+   lineman build
+   cd -
+}
+
 if [ $1 == "public" ]; then
    exec uwsgi --ini /app/uwsgi/public.ini --python-autoreload 1
 elif [ $1 == "admin" ]; then
+   build_front_end
    exec uwsgi --ini /app/uwsgi/admin.ini --python-autoreload 1
 elif [ $1 == "admin-dev" ]; then
+   build_front_end
```

Comment:
I initially put the build step in `Dockerfile`. It turned out it broke the dev environment because the volumes binding occurs after building. There may be another build script that I missed. In this case, I'd be happy to move it there 😃